tests/10apidoc/03events-initial.pl

```diff
@@ -226,10 +226,6 @@ sub matrix_sync
    )->on_done( sub {
       my ( $body ) = @_;
 
-      assert_json_keys( $body, qw( account_data rooms presence next_batch ) );
```

Comment:
next_batch is used in line 234/230 doesn't this cause a perl key error if perl has something like that?

Reply:
Oops, you're right. next_batch is even required. Can I do `assert_json_keys( $body, qw( next_batch ) );` ? (I'm not sure what qw does - is it for exact matching?)

Reply:
Pushed ^, I think it's correct

Reply:
according to google ;) qw just quotes words. So that should work